timestamp,category,user_id,function,line_number,flag,message,exception
2024-06-08 13:55:32,ERROR,asif24,create_or_pass_folder,197,0,Function create_or_pass_folder error,"Function create_or_pass_folder error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 190, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:4909abe3-b01e-004d-4e7d-b9b3e4000000
Time:2024-06-08T08:25:32.5022624Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:4909abe3-b01e-004d-4e7d-b9b3e4000000
Time:2024-06-08T08:25:32.5022624Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error>"
2024-06-08 13:55:32,ERROR,asif24,create_or_pass_folder,206,0,"Function create_or_pass_folder Error creating folder'asif24': Value for one of the query parameters specified in the request URI is invalid.
RequestId:4909abe3-b01e-004d-4e7d-b9b3e4000000
Time:2024-06-08T08:25:32.5022624Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:4909abe3-b01e-004d-4e7d-b9b3e4000000
Time:2024-06-08T08:25:32.5022624Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error> ","Function create_or_pass_folder Error creating folder'asif24': Value for one of the query parameters specified in the request URI is invalid.
RequestId:4909abe3-b01e-004d-4e7d-b9b3e4000000
Time:2024-06-08T08:25:32.5022624Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:4909abe3-b01e-004d-4e7d-b9b3e4000000
Time:2024-06-08T08:25:32.5022624Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error> 
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 190, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:4909abe3-b01e-004d-4e7d-b9b3e4000000
Time:2024-06-08T08:25:32.5022624Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:4909abe3-b01e-004d-4e7d-b9b3e4000000
Time:2024-06-08T08:25:32.5022624Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error>"
2024-06-08 13:55:32,INFO,asif24,home,1403,1,User asif24 logged in successfully,
2024-06-08 13:55:33,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 13:55:33,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 13:55:35,INFO,asif24,get_data_source,1993,1,table_update route successfully send data,
2024-06-08 13:55:38,INFO,asif24,Cogni_button,1710,1,CogniLink load button pressed,
2024-06-08 13:55:40,ERROR,asif24,update_when_file_delete,611,0,Function update_when_file_delete error,"Function update_when_file_delete error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 418, in update_when_file_delete
    blob_client = blob_service_client.get_blob_client(container=container_name, blob=blob.name)
AttributeError: 'str' object has no attribute 'name'"
2024-06-08 13:55:40,INFO,asif24,Cogni_button,1722,1,Load Data Function executed in 2.7882843017578125 seconds,
2024-06-08 13:55:42,INFO,asif24,get_data_source,1993,1,table_update route successfully send data,
